openapi.yaml

```diff
@@ -960,3 +997,109 @@ components:
         approvingTeamDetails:
           type: string
           example: "DevRel"
+    SchemaRequest:
+      title: SchemaRequest
+      required:
+        - schemafull
+      type: object
+      properties:
+        req_no:
+          title: req_no
+          type: integer
+          format: int32
```

Comment:
Question: I'm guessing we don't really need to provide examples for such properties (integers). Maybe we should anyway for consistency?

Reply:
I feel it's good to have it for consistency, but I've to say that in some props I couldn't add examples because I didn't know how it should look like (I mean, a string is a string, but I would like to have examples that are near to what it would be in real live) :D

Reply:
Yes, I think it's good to have a chat with Murali when we don't know what something is ^^ Really helped on my last PR